scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/IntegrationFailureException.java

```diff
@@ -0,0 +1,17 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+public abstract /*sealed*/ class IntegrationFailureException extends RuntimeException {
```

Comment:
Can uncomment sealed and add `permits DatabaseException, FailedInsertException` before the `extends` keyword :)

Or you can feel free to remove the `sealed` comment altogether, but the `sealed`/`permits` pattern may be nice to have if we ever want to do some sort of double-dispatch (`instanceof` pattern matching) on a plain `IntegrationFailureException` type.

Reply:
Alright, this should be fixed now. I've never worked with sealed anything before, so thanks for explaining how to implement it to me!

Reply:
@pcrosemurgy Just pinging you to resolve or comment on this.